src/domain/resource/utils.js

```diff
@@ -434,7 +434,11 @@ export const isTimeRangeReservable = (resource, start, end) => {
  * @param resource {object} Resource object.
  * @returns {boolean}
  */
-export const isFullCalendarEventDurationEditable = (resource) => {
+export const isFullCalendarEventDurationEditable = (resource, start, end, isStaff = false) => {
+  if (isStaff) {
+    return true;
```

Comment:
Same comment here

Reply:
I added the code comment to the function dockblock , that should be more correct right?